packages/ens-deployments/package.json

```diff
@@ -12,23 +12,32 @@
   "homepage": "https://github.com/namehash/ensnode/tree/main/packages/ens-deployments",
   "keywords": ["ENS", "ENSNode"],
   "exports": "./src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "exports": {
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "main": "./dist/index.js",
+    "module": "./dist/index.mjs",
+    "types": "./dist/index.d.ts"
+  },
   "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
```

Comment:
Do we need the `dev` script for release? Or is that for debugging only?

Reply:
We don't need it, but it has been useful in the past, for example when using `pnpm link` to a project outside the monorepo. We don't necessarily need it now, so happy to remove.